client/src/components/CreateContestForm.js

```diff
@@ -71,7 +71,7 @@ export default function CreateContestForm() {
     const [date, setDate] = useState(new Date().toISOString().substr(0,10))
     const [time, setTime] = useState(new Date().toTimeString().substr(0,5))
     const [timeZone, setTimeZone] = useState('PDT')
-    const [images, setImages] = useState('')
+    const [inspirationalImages, setImages] = useState(new Array())
```

Comment:
```suggestion
    const [inspirationalImages, setImages] = useState([])
```

you can simple just initiate an empty array :)